core/src/main/java/com/scalar/db/service/TransactionFactory.java

```diff
@@ -50,8 +51,10 @@ public DistributedTransactionAdmin getTransactionAdmin() {
   /**
    * Returns a {@link TwoPhaseCommitTransactionManager} instance
    *
-   * @return a {@link TwoPhaseCommitTransactionManager} instance
+   * @return a {@link TwoPhaseCommitTransactionManager} instance. If the transaction manager does
+   *     not support the two-phase commit interface, returns {@code null}.
    */
+  @Nullable
   public TwoPhaseCommitTransactionManager getTwoPhaseCommitTransactionManager() {
     return ProviderManager.createTwoPhaseCommitTransactionManager(config);
   }
```

Comment:
Made this method return null if the transaction manager doesn't support the two-phase commit interface.